portable-atomic-util/src/arc.rs

```diff
@@ -2234,7 +2244,7 @@ impl<T> Default for Arc<[T]> {
     #[inline]
     fn default() -> Self {
         // TODO: we cannot use non-allocation optimization (https://github.com/rust-lang/rust/blob/1.80.0/library/alloc/src/sync.rs#L3449)
-        // for now due to casting Arc<[T; N]> -> Arc<[T]> requires unstable CoerceUnsized.
+        // for now since casting Arc<[T; N]> -> Arc<[T]> requires unstable CoerceUnsized from Rust nightly.
```

Comment:
How about I remove "from nightly" & keep the other change here like this?

```suggestion
        // for now since casting Arc<[T; N]> -> Arc<[T]> requires unstable CoerceUnsized.
```

Reply:
> How about I remove "from nightly" & keep the other change here like this?

Seems reasonable to me.